tests/create-tests.sh

```diff
@@ -365,7 +368,12 @@ lzip -k -o simple{.lzip,}  # Default extension: .lz
 lzop -k simple  # extension .lzo
 compress < simple > simple.Z
 
-wget 'https://raw.githubusercontent.com/iipc/warc-specifications/master/primers/web-archive-formats/hello-world.warc'
+if [ -n "${HELLO_WORLD_WARC_FILE:-}" ]; then
+    # offline
+    cp "$HELLO_WORLD_WARC_FILE" hello-world.warc
+else
+    wget 'https://raw.githubusercontent.com/iipc/warc-specifications/master/primers/web-archive-formats/hello-world.warc'
+fi
```

Comment:
This seems simpler:

```suggestion
if [[ ! -f hello-world.warc ]; then
    wget 'https://raw.githubusercontent.com/iipc/warc-specifications/master/primers/web-archive-formats/hello-world.warc'
fi
```

Would that work for you?